pkg/util/mount/mount_linux.go

```diff
@@ -396,9 +394,8 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 // diskLooksUnformatted uses 'lsblk' to see if the given disk is unformated
 func (mounter *SafeFormatAndMount) getDiskFormat(disk string) (string, error) {
 	args := []string{"-n", "-o", "FSTYPE", disk}
-	cmd := mounter.Runner.Command("lsblk", args...)
 	glog.V(4).Infof("Attempting to determine if disk %q is formatted using lsblk with args: (%v)", disk, args)
-	dataOut, err := cmd.CombinedOutput()
+	dataOut, err := mounter.Exec.Run("lsblk", args...)
```

Comment:
I'm not sure what you mean by this comment. `findmnt` is not used by `mount_linux.go` and it has no use when finding a fs on a block device.